web-api/openapi/schemas/document.yml

```diff
@@ -11,29 +11,32 @@ DocumentPropertyId:
 DocumentPropertyString:
   description: |
     Arbitrary string property that can be attached to a document.
-    Enclosing whitespace will be trimmed.
     The length constraints are in bytes, not characters.
   type: string
   pattern: '^[^\x00]+$'
-  minLength: 1
+  minLength: 0
```

Comment:
❓ why should we accept empty strings at all? if we get an empty string, this is most likely a logic error on the customer side. and it also doesn't bring any benefit to our filter logic, on the contrary.

❓ also, why shouldn't we trim them, or if we don't want to change the input, why don't we at least validate if they are trimmed?